templates/docs/patterns/icons/index.md

```diff
@@ -164,6 +170,8 @@ If you use a limited set of icons you may want to include them individually to r
 // additional icons
 {% for icon in additional_icons %}@include vf-p-icon-{{ icon }};
 {% endfor %}
+{% for icon in additional_icons_new %}@include vf-p-icon-{{ icon }};
+{% endfor %}
```

Comment:
Why not concatenate `additional_icons` and `additional_icons_new` before printing their `@include` statements? Is it to intentionally place a line break between the old and new additional icons?

Maybe it would be clearer to put them all in one alphabetized block of additional icons imports, so that the new ones don't appear to be "second-class" or an afterthought.

Reply:
Now that I re-read your updated docs, it seems this is intentionally done to separate the new icons from the old icons, to make it easier for someone to understand what has changed. I think this is OK for v4, and they should be merged together in v5.

Reply:
Yes, I had them merged (and alphabetised) first, but then I realised it's impossible to tell which ones are new.

I think it would be OK to keep them separated now, and eventually merge them after they settle in a bit.